server/backend/movie.go

```diff
@@ -0,0 +1,43 @@
+package backend
+
+import (
+	"context"
+
+	pb "github.com/TUM-Dev/Campus-Backend/server/api/tumdev"
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func (s *CampusServer) GetMovies(_ context.Context, req *pb.GetMoviesRequest) (*pb.GetMoviesReply, error) {
+	var movies []model.Kino
+	if err := s.db.Joins("Files").Find(&movies, "kino > ?", req.LastId).Error; err != nil {
```

Comment:
Probably it would be good to propergate the context here: https://gorm.io/docs/context.html